### PR TITLE
Pytorch1.7.

### PR DESCRIPTION
**Pull request by atselousov**
Updated `pytorch` to version 1.7

### CODE REVIEW

File: targets/python37-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+# python        3.8       (apt)
 # jupyter       latest    (pip)
 # jupyterlab    latest    (pip)
-# pytorch       1.6.0     (docker-hub)
+# pytorch       1.7.0     (docker-hub)
 # tensorflow    2.3.0     (pip)
 # ==================================================================
 
-FROM pytorch/pytorch:1.6.0-cuda10.1-cudnn7-devel
+FROM pytorch/pytorch:1.7.0-cuda11.0-cudnn8-devel
```
Comment (yartem) [inline targets/python37-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile:11]: why do we use `devel`? `runtime` is 2G comparing to devel's 5G. See the [difference](https://stackoverflow.com/a/59168833/4977823).

File: targets/python37-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile [hunk 2]
```diff
@@ -1,14 +1,14 @@
 # ==================================================================
 # module list
 # ------------------------------------------------------------------
-# python        3.7       (apt)
+# python        3.8       (apt)
```
Comment (yartem) [inline targets/python37-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile:4]: by the way: we could re-install python manually if 3.8 is too new for us

File: targets/python37-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile (hunk 1 above)
Comment (atselousov) [reply targets/python37-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile:11]: As I know, `runtime` image does not provide nvcc. So, you can build `apex`, it's just one lib that requires `nvcc`.

File: targets/python37-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile (hunk 2 above)
Comment (atselousov) [reply targets/python37-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile:4]: Is `python3.8` still not supported?

**Comment by YevheniiSemendiak** [comment]
done in #145

**Comment by YevheniiSemendiak** [comment]
done in #145

**Pull request closed without merging**